torch/_utils.py

```diff
@@ -185,6 +185,12 @@ def _rebuild_mlc_tensor(data, dtype, device, requires_grad):
     return tensor
 
 
+def _rebuild_ort_tensor(data, dtype, device, requires_grad):
```

Comment:
Why not just re-use the function from above?

Reply:
Copy pasta because no one else decided to reuse previous copies of the same logic either (looking at `_rebuild_mlc_tensor` and `_rebuild_xla_tensor`).

I'd be happy to refactor the three helper functions, but I generally go for "be as least disruptive as possible" and "follow the coding conventions closest to your change" on codebases to which I am not a primary contributor, which is what I did here 😄

Reply:
I've added a patch to this PR that consolidates these three identical functions just in case we want to do that now. If not I can drop the patch or open a subsequent PR.